package.json

```diff
@@ -16,7 +16,7 @@
     "pretest:integration": "pnpm build",
     "test:integration": "pnpm playwright:integration",
     "posttest:integration": "pnpm clean:integration",
-    "playwright:integration": "playwright test --config ./integration/playwright.config.ts --project chromium",
+    "playwright:integration": "playwright test --config ./integration/playwright.config.ts --project=chromium",
```

Comment:
I changed this so that I can pass a test name, otherwise it treats it as another project name.